widgetssdk/src/main/java/com/glia/widgets/chat/adapter/holder/fileattachment/VisitorFileAttachmentViewHolder.java

```diff
@@ -34,5 +38,28 @@ private void setupDeliveredView(Context context, UiTheme uiTheme) {
     public void bind(VisitorAttachmentItem item) {
         super.setData(item.isFileExists, item.isDownloading, item.attachmentFile);
         deliveredView.setVisibility(item.showDelivered ? View.VISIBLE : View.GONE);
+
+        String name = item.attachmentFile.getName();
```

Comment:
could be a separate method to improve readability, same for `setAccessibilityDelegate`

Reply:
fixed